flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/MesosResourceManager.java

```diff
@@ -120,6 +120,10 @@
 	/** A local actor system for using the helper actors. */
 	private final ActorSystem actorSystem;
 
+	/** Web url for to show in mesos page.*/
```

Comment:
Nitpicking here but there should be a space between the period and the asterisk for consistency:
```
	/** Web url for to show in mesos page. */
```